tests/admin_views/admin.py

```diff
@@ -156,6 +156,10 @@ def has_change_permission(self, request, obj=None):
         """ Only allow changing objects with even id number """
         return request.user.is_staff and (obj is not None) and (obj.id % 2 == 0)
 
+    def has_view_permission(self, request, obj=None):
+        """ Only allow viewing objects with id number multiple of 3 """
+        return request.user.is_staff and (obj is not None) and (obj.id % 3 == 0)
```

Comment:
The parentheses on this line are unnecessary.

Reply:
Ok (that was copied from just above, I'm never sure if it's better to keep resemblance with existing code for consistency or do things right)